### PR TITLE
Add data to db

### PR DESCRIPTION
**Pull request by diego-centella-jalafund**

### CODE REVIEW

File: src/Data/Scripts/DBData.sql [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+    localization_id UUID NOT NULL,
+    name VARCHAR NOT NULL,
+    emailAddress VARCHAR NOT NULL,
+    categoryId UUID NOT NULL,
+    workScheduleId UUID NOT NULL
+);
+
+CREATE TABLE IF NOT EXISTS jobs (
+    id UUID PRIMARY KEY,
+    user_client_id UUID NOT NULL,
```
Comment (SebasthianSalpozz) [inline src/Data/Scripts/DBData.sql:35]: I think that is not necessary why is the reason to have a Client Id

**Review by SebasthianSalpozz — APPROVED** [review]

File: src/Data/Scripts/DBData.sql (hunk 1 above)
Comment (SebasthianSalpozz) [inline src/Data/Scripts/DBData.sql:35]: That exist in the jobs data base ?

**Review by SebasthianSalpozz — CHANGES_REQUESTED** [review]

File: src/Data/Scripts/DBData.sql (hunk 1 above)
Comment (diego-centella-jalafund) [reply src/Data/Scripts/DBData.sql:35]: The row user_client_id exists in the jobs data base

Comment (diego-centella-jalafund) [reply src/Data/Scripts/DBData.sql:35]: The client Id is required on the jobs data base

Comment (SebasthianSalpozz) [reply src/Data/Scripts/DBData.sql:35]: ohhh ok thanks my bada

Comment (SebasthianSalpozz) [reply src/Data/Scripts/DBData.sql:35]: ohhh ok thanks my bada

**Review by SebasthianSalpozz — APPROVED** [review]

**Pull request merged**